app/policies/schools/standing_policy.rb

```diff
@@ -0,0 +1,12 @@
+module Schools
+  class StandingPolicy < ApplicationPolicy
+    def new?
+      user&.school_admin.present? && user.school == current_school
+    end
```

Comment:
The user will always be present in this case. 
```suggestion
    def new?
      current_school_admin.present? && user.school == current_school
    end
```